### ISSUE TITLE
Email Sending Option

### ISSUE THREAD

**Issue by hassanrehman01398** [mit-cml/appinventor-sources#2105]
I wish we could work on sending email directly form our mit app as it proceed to gmail first then it sends email.

**Comment by shaurya0806** [comment]
can i work on this

**Comment by hassanrehman01398** [comment]
Sure. We both can if you want.

**Comment by manavroX** [comment]
can i work on this too?

**Comment by yashsrv** [comment]
Hi @ewpatton , I want to work on this feature. Should I create a new component for this? If yes, then under which category? If not, then under which existing component?

**Comment by ewpatton** [comment]
Yes, I would say it should go under the Social category alongside components like PhoneCall and Texting.

**Comment by yashsrv** [comment]
@ewpatton Got it! Regarding implementation of it, should it use Android’s intent to hand off the email to an email client?

**Comment by yashsrv** [comment]
> Yes, I would say it should go under the Social category alongside components like PhoneCall and Texting.

@ewpatton Hello, I've opened a PR https://github.com/mit-cml/appinventor-sources/pull/3942

It's ready for review whenever you get a chance
